wagtail/admin/templates/wagtailadmin/account/settings_panels/avatar.html

```diff
@@ -1,35 +1,16 @@
 {% load i18n wagtailadmin_tags %}
 {% block content %}
-    <div class="avatar-panel">
-        <div class="avatar-panel__image">
-            {% avatar user=request.user size="large" %}
+    <div class="avatar-panel w-flex w-flex-wrap w-items-center w-mt-5">
+        <div class="avatar-panel__image w-mr-9 w-mb-5">
+            {% if form.avatar.value %}
+                <input class="avatar-clear" type="checkbox" name="avatar-avatar-clear" id="avatar-avatar-clear_id">{% trans "Reset to default" %}
+                <img class="avatar large gravatar w-block w-mt-3" src="{% avatar_url request.user 70 gravatar_only=True %}" alt="default avatar image"/>
```

Comment:
Note - we should always use translations for any text, even screen reader text.

```suggestion
                <img class="avatar large gravatar w-block w-mt-3" src="{% avatar_url request.user 70 gravatar_only=True %}" alt="{% trans "Default avatar image" %}/>
```

Reply:
Alright 👍

Reply:
Note I ended up going with `Default avatar` as there is no need to repeat the word image.
Screenshot of screen reader before the change.

_Chrome with voiceover on mac_
<img width="660" alt="Screenshot 2023-02-07 at 8 34 21 pm" src="https://user-images.githubusercontent.com/1396140/217221712-d7498236-f343-4886-b546-2afc0d9cce01.png">